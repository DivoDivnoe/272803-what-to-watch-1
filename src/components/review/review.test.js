import React from 'react';
import renderer from 'react-test-renderer';
import Review from './review.jsx';

const mock = {
  review: {
    id: 11,
    comment: `Yohoho`,
    date: ``,
    user: {
      id: 2,
      name: `user`,
    }
  },
};

describe(`Review component`, () => {
  it(`renders correctly`, () => {
    const tree = renderer.create(
        <Review
          review={mock.review}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
