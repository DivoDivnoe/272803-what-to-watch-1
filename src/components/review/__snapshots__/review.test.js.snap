// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Review component renders correctly 1`] = `
<div
  className="review"
>
  <blockquote
    className="review__quote"
  >
    <p
      className="review__text"
    >
      Yohoho
    </p>
    <footer
      className="review__details"
    >
      <cite
        className="review__author"
      >
        user
      </cite>
      <time
        className="review__date"
        dateTime=""
      >
        Invalid Date NaN, NaN
      </time>
    </footer>
  </blockquote>
  <div
    className="review__rating"
  />
</div>
`;
